scripts/quests/crystalWar/A_Little_Knowledge.lua

```diff
@@ -0,0 +1,143 @@
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.CRYSTAL_WAR, xi.quest.id.crystalWar.A_LITTLE_KNOWLEDGE)
+
+quest.reward =
+{
+    ki = xi.ki.GRIMOIRE,
+    title = xi.title.SCHULTZ_SCHOLAR,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and player:getMainLvl() >= xi.settings.main.ADVANCED_JOB_LEVEL
+        end,
+
+        [xi.zone.THE_ELDIEME_NECROPOLIS_S] =
+        {
+            ['Erlene'] =
+            {
+                onTrigger = function(player, npc)
+                    return quest:progressEvent(10, 1)
+                end
+            },
+
+            onEventFinish =
+            {
+                [10] = function(player, csid, option, npc)
+                    quest:begin(player)
+                end,
+            }
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED
+        end,
+
+        [xi.zone.THE_ELDIEME_NECROPOLIS_S] =
+        {
+            ['Erlene'] =
+            {
+                onTrade = function(player, npc, trade)
+                    local mJob = player:getMainJob()
+                    if npcUtil.tradeHasExactly(trade, { { xi.items.VELLUM, 12 } }) then
+                        player:tradeComplete()
+                        if
+                            mJob == xi.job.BLM or
+                            mJob == xi.job.RDM or
+                            mJob == xi.job.SMN or
+                            mJob == xi.job.BLU
+                        then
+                            return quest:progressEvent(12, 1)
+                        else
+                            return quest:progressEvent(12)
+                        end
+                    end
+                end,
+
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Status') == 0 then
+                        return quest:progressEvent(11)
+                    elseif quest:getVar(player, 'Status') == 1 then
+                        if
+                            player:hasStatusEffect(xi.effect.MANAFONT) or
+                            player:hasStatusEffect(xi.effect.CHAINSPELL) or
+                            player:hasStatusEffect(xi.effect.ASTRAL_FLOW) or
+                            player:hasStatusEffect(xi.effect.AZURE_LORE)
+                        then
+                            return quest:progressEvent(14)
+                        else
+                            return quest:progressEvent(13)
+                        end
+                    end
+                end
+            },
+
+            onEventFinish =
+            {
+                [12] = function(player, csid, option, npc)
+                    quest:setVar(player, 'Status', 1)
+                end,
+
+                [14] = function(player, csid, option, npc)
+                    local ID = zones[xi.zone.THE_ELDIEME_NECROPOLIS_S]
```

Comment:
Unnecessary, since this only occurs here.  Define local eldiemeID = require(...) at the top of the file